### PR TITLE
Add test for TC with arbitrary newlines

### PR DESCRIPTION
**Pull request by nicolasvasilache**

### CODE REVIEW

File: test/cuda/test_corner_cases.cc [hunk 1]
```diff
@@ -282,6 +282,38 @@ TEST(TestCornerCases, E23) {
       at::Scalar(d[0]).toFloat());
 }
 
+TEST(TestCornerCases, E24) {
```
Comment (skimo-openhub) [inline test/cuda/test_corner_cases.cc]: Please also explain what corner case is being tested in a comment.

**Comment by nicolasvasilache** [comment]
@caffe2bot retest this please

File: test/cuda/test_corner_cases.cc [hunk 2]
```diff
@@ -282,6 +282,44 @@ TEST(TestCornerCases, E23) {
       at::Scalar(d[0]).toFloat());
 }
 
+// This tests that the TC parser functions in the presence of arbitrary newlines
```
Comment (ftynse) [inline test/cuda/test_corner_cases.cc]: parser -> _parses_

**Comment by skimo-openhub** [comment]
On Mon, May 07, 2018 at 08:59:57PM +0000, ftynse wrote:
> ftynse approved this pull request.
> > @@ -282,6 +282,44 @@ TEST(TestCornerCases, E23) {
>        at::Scalar(d[0]).toFloat());
>  }
>  
> +// This tests that the TC parser functions in the presence of arbitrary newlines
> 
> parser -> _parses_

I think the original is correct (where "functions" is a verb).
If you take "functions" to be a noun, then you'd also have to drop
"the" in front of "TC".

skimo
